sdk/openai/Azure.AI.OpenAI/src/Custom/OpenAIClient.cs

```diff
@@ -17,40 +17,256 @@ namespace Azure.AI.OpenAI
     /// <summary> Azure OpenAI APIs for completions and search. </summary>
     public partial class OpenAIClient
     {
+        private const string publicOpenAIVersion = "1";
+        private const string publicOpenAIEndpoint = $"https://api.openai.com/v{publicOpenAIVersion}";
+
+        /// <remarks> This is the deployment id for all Azure OpenAI operations (completions and embeddings) </remarks>
+        private string DeploymentId { get; }
+
+        /// <remarks> This key is used to connect to a non-Azure OpenAI resource created directly with OpenAI.
+        /// For Azure OpenAI resources as created and maintained in Azure Portal, use a constructor that provides
+        /// an Azure resource endpoint and credential, instead. </remarks>
+        private string PublicOpenAIToken { get; }
+
+        /// <summary> Initializes a instance of OpenAIClient using the public OpenAI endpoint. </summary>
+        /// <param name="openAIAuthToken "> A credential used to authenticate to an Azure Service. </param>
```

Comment:
The `<param>` says "A credential used to authenticate to an Azure Service", but my understanding is that this constructor is actually for non-Azure scenarios. Does this need to be changed?

Reply:
Good catch, will update.